platform.json

```diff
@@ -17,7 +17,8 @@
     "https://dl.bintray.com/platformio/dl-packages/manifest.json",
     "https://sourceforge.net/projects/platformio-storage/files/packages/manifest.json/download",
     "http://dl.platformio.org/packages/manifest.json",
-    "https://raw.githubusercontent.com/eerimoq/simba/master/make/platformio/manifest.json"
+    "https://raw.githubusercontent.com/eerimoq/simba/master/make/platformio/manifest.json",
+    "https://raw.githubusercontent.com/eerimoq/pumbaa/master/make/platformio/manifest.json"
```

Comment:
Dou you want to merge it now? I have not made a release of this platform yet. Manifest file can contain multiple packages.